### ISSUE TITLE
[pni] category: thumbnails are missing "alt" attribute

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4644]
Our product thumbnails do not have an alt attribute, I don't even know how that happened O_o
